src/config/env.ts

```diff
@@ -7,11 +7,25 @@ const env = dotenv.config({
   path: path.resolve(process.cwd(), filename),
 });
 
+function getProxyFromEnvironment(): string{
+  let proxyHost: string = process.env.EXTERNAL_ONLY_PROXY_HOST;
+  let proxyPort: string = process.env.EXTERNAL_ONLY_PROXY_PORT;
+
+  if(!proxyHost || !proxyPort){
+    return '';
+  }
+  const proxyAddress = `http://${proxyHost}:${proxyPort}`;
+
+  return proxyAddress;
+}
+
 export default {
   MICROS_ENV: process.env.MICROS_ENV || 'development',
   MICROS_SERVICE_VERSION: process.env.MICROS_SERVICE_VERSION,
   NODE_ENV: process.env.NODE_ENV,
   SENTRY_DSN: process.env.SENTRY_DSN,
+  // The proxy to use for outbound calls to GitHub. Leave empty to not use a proxy.
+  GITHUB_PROXY: getProxyFromEnvironment(),
```

Comment:
Would probably be good to change this to `GLOBAL_PROXY`

Reply:
Why? It's explicitly a proxy only for calls to GitHub. But I'll rename it to `GITHUB_API_PROXY` instead, I guess, to make it more meaningful.